app/src/androidTest/java/net/skyscanner/backpack/docs/DocsRegistry.kt

```diff
@@ -89,16 +88,12 @@ object DocsRegistry {
     ViewScreenshot("Dialog - View", "with-cta") { setupDialog("Success Three Buttons") },
     ViewScreenshot("Dialog - View", "delete-confirmation") { setupDialog("Destructive") },
     ViewScreenshot("Dialog - View", "with-flare") { setupDialog("Flare") },
-    ComposeScreenshot("Dialog - Compose", "success") { setupComposeDialog(it, ShownDialog.SuccessThreeButtons) },
-    ComposeScreenshot("Dialog - Compose", "warning") { setupComposeDialog(it, ShownDialog.Warning) },
-    ComposeScreenshot("Dialog - Compose", "destructive") { setupComposeDialog(it, ShownDialog.Destructive) },
-    ComposeScreenshot("Dialog - Compose", "flare") { setupComposeDialog(it, ShownDialog.Flare) },
-    ComposeScreenshot("Dialog - Compose", "image-start-alignment") {
-      setupComposeDialog(it, ShownDialog.ImageStartAlignment)
-    },
-    ComposeScreenshot("Dialog - Compose", "image-end-alignment") {
-      setupComposeDialog(it, ShownDialog.ImageEndAlignment)
-    },
+    ComposeScreenshot("Dialog - Compose", "success"),
```

Comment:
something snuck in here?

Reply:
Yes, a part of the screenshots experiment! Good catch :D